OrbitCore/Capture.cpp

```diff
@@ -423,7 +423,7 @@ bool Capture::IsCapturing() {
 //-----------------------------------------------------------------------------
 TcpEntity* Capture::GetMainTcpEntity() {
   return Capture::IsRemote() ? (TcpEntity*)GTcpClient.get()
-                             : (TcpEntity*)GTcpServer;
+                             : (TcpEntity*)GTcpServer.get();
```

Comment:
nit: It seems like the cast is not needed here.

Reply:
done